.github/workflows/ci-backend.yml

```diff
@@ -265,17 +265,34 @@ jobs:
                   person-on-events: ${{ matrix.person-on-events }}
 
             - uses: EndBug/add-and-commit@v9 # You can change this to use a specific version.
+              id: add-and-commit
               if: ${{ matrix.clickhouse-server-image == 'clickhouse/clickhouse-server:22.3' && !matrix.person-on-events }}
               with:
                   # The arguments for the `git add` command (see the paragraph below for more info)
                   # Default: '.'
-                  add: '["ee/clickhouse", "posthog/api/test/__snapshots__", "posthog/test/__snapshots__"]'
+                  add: '["ee/clickhouse", "posthog/api/test/__snapshots__", "posthog/test/__snapshots__", "posthog/queries/"]'
 
                   default_author: github_actions
 
                   # The message for the commit.
                   # Default: 'Commit from GitHub Actions (name of the workflow)'
                   message: 'Update snapshots'
+            - name: Check if any snapshot changes were left uncomitted
+              id: changed-files
+              run: |
+                  if [[ -z $(git status -s | grep -v ".test_durations" | tr -d "\n") ]]
+                  then
+                    echo '::set-output name=diff::$(git status --porcelain)'
+                    echo '::set-output name=outcome::success'
+                  else
+                    echo '::set-output name=outcome::failure'
+                  fi
+
+            - name: Fail CI if some snapshots have been updated but not committed
+              if: steps.changed-files.outputs.outcome == 'success' && steps.add-and-commit.outcome == 'success'
```

Comment:
NIT: Shouldn't we label this as a "failure"? Seems counterintuitive to read two successes and exit

Reply:
lol, fair!

How about something like:

`outputs.files_found == 'true'` and then we fail when files are found

Reply:
yup